wordpress-mobile/WordPress-Android
# [Site Creation] Tweak UI of Site Theme to match design specs

Tweaks the UI of the Site Theme screen to match the Figma Designs:
- Add divider on top of helper text footer
- Increase padding around helper text in footer to `16dp`
- Increase font size of category title to `20sp`

Small refactoring was also added to this PR, concerning code cleanup that we identified after merging #16584

To test:
1. Start Site Creation flow
2. Skip until the `Choose a design/theme` screen
3. **Expect** UI changes:
   - Font size of category titles increased
   - Padding around footer text slightly increased
   - Semi-transparent divider between last preview cards and footer text

### Previews:

| Category Titles Before | Category Titles After |
| --- | --- |
| ![fs_site_before](https://user-images.githubusercontent.com/4588074/169813491-771b257c-788b-4ad4-9b55-1a42633e5ebf.png) | ![fs_site_after](https://user-images.githubusercontent.com/4588074/169813512-29d8ae75-2d3c-45ac-95e6-1f030c448106.png) |

| Footer Before | Footer After |
| --- | --- |
|![footer_before](https://user-images.githubusercontent.com/4588074/169813348-f24e6fb5-0e58-48ac-83f5-207b60566f2f.png) | ![footer_after](https://user-images.githubusercontent.com/4588074/169813252-e562e63a-64b1-4ea3-839b-4310a343ec0b.png) |

## Regression Notes
1. Potential unintended areas of impact
   - Category titles font size on the Page layout picker (FAB on My Site → Site page)

5. What I did to test those areas of impact (or what existing automated tests I relied on)
   - Manual testing.

6. What automated tests I added (or what prevented me from doing so)
   - Non-functional UI changes are not in scope of automated tests.

PR submission checklist:
